openedx/features/job_board/models.py

```diff
@@ -1,31 +1,49 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.core.validators import FileExtensionValidator
 from django.db import models
 from django_countries.fields import CountryField
 from model_utils.models import TimeStampedModel
 
-from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES
+from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES, LOGO_ALLOWED_EXTENSION
+from .helpers import validate_file_size
 
 
 class Job(TimeStampedModel):
     """
     This model contains all the fields related to a job being
     posted on the job board.
     """
-    title = models.CharField(max_length=255)
-    company = models.CharField(max_length=255)
-    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES)
-    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES)
-    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES)
-    city = models.CharField(max_length=255)
-    country = CountryField()
-    description = models.TextField()
-    function = models.TextField(blank=True, null=True)
-    responsibilities = models.TextField(blank=True, null=True)
-    website_link = models.URLField(max_length=255, blank=True, null=True)
-    contact_email = models.EmailField(max_length=255)
-    logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True)
+    title = models.CharField(max_length=255, verbose_name='Job Title')
+    company = models.CharField(max_length=255, verbose_name='Organization Name')
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES, blank=False, verbose_name='Job Type',
+                            default=0, help_text='Please select whether the job is onsite or can be done remotely.')
+    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES, verbose_name='Compensation',
```

Comment:
Should we use translation string for `verbose_name` and `help_text`? We can use `gettext_lazy` version as django does with its [models](https://github.com/django/django/blob/master/django/contrib/auth/models.py#L304)

Reply:
done